packages/coordinator/src/screens/bookedDonation/BookedAppointmentScreen.tsx

```diff
@@ -33,7 +41,30 @@ export default function BookedAppointmentScreen(
         stickyComponent: <div>{props.appointment.firstName}</div>,
       }}
     >
-      {JSON.stringify(props)}
+      <div className={styles.details}>פרטי התורם</div>
+      <InfoBar
+        title={"מספר טלפון"}
+        value={
+          <a href={`tel: ${props.appointment.phone}`}>
+            {props.appointment.phone}
+          </a>
+        }
+        icon={<Phone />}
+      />
+      <InfoBar
+        title={"סוג דם"}
+        value={getValueInDiv(props.appointment.bloodType)}
+        icon={<Bloodtype />}
+      />
+      <InfoBar
+        title={"נקבע בתאריך"}
+        value={getValueInDiv(bookingTime)}
+        icon={<Calender />}
+      />
     </CoordinatorScreen>
   );
 }
+
+function getValueInDiv(value: string) {
+  return <div className={styles.values}>{value}</div>;
```

Comment:
Since all this does is adding bold to the value, you could wrap the InfoBar value in this div and remove duplication.

What do you think?